tests/tests/optionsTests.robot

```diff
@@ -114,7 +114,8 @@ Enable Debug Output functionality test
 Verify Output log to a file option
     [Documentation]    Verify Output logs to a file option
     [Tags]    options
-    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${file_4mb}[archive_name].zip
+
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${extraction_filter}[archive_name].zip
```

Comment:
You changed variable here, but in line 124 there is still ${file_4mb} (also uses "Ubuntu Upload Test File" instead of "Upload Test File"